src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -488,18 +488,16 @@ var DynamicTable = new Class({
             this.selectedRows.empty();
         },
 
-        selectRow : function (rowId) {
-            this.selectedRows.empty();
+        selectRow : function (rowId, clear) {
+            if (clear !== false)
+                this.selectedRows.empty();
             this.selectedRows.push(rowId);
             var trs = this.tableBody.getElements('tr');
             for (var i = 0; i < trs.length; i++) {
                 var tr = trs[i];
-                if (tr.rowId == rowId) {
-                    if (!tr.hasClass('selected'))
-                        tr.addClass('selected');
-                }
-                else
-                if (tr.hasClass('selected'))
+                if ((tr.rowId == rowId) && (!tr.hasClass('selected')))
+                    tr.addClass('selected');
+                else if ((clear !== false) && (tr.hasClass('selected')))
```

Comment:
`tr.hasClass('selected')` check is redundant?

Reply:
In that case, `!tr.hasClass('selected')` two lines above is redundant as well. I shall remove both.